core/client/verify.go

```diff
@@ -37,21 +40,42 @@ var (
 // Verifier is a client helper library for verifying request and responses.
 type Verifier struct {
 	vrf         vrf.PublicKey
-	mapVerifier *client.MapVerifier
-	logVerifier client.LogVerifier
+	mapVerifier *tclient.MapVerifier
+	logVerifier tclient.LogVerifier
 }
 
 // NewVerifier creates a new instance of the client verifier.
 func NewVerifier(vrf vrf.PublicKey,
-	mapVerifier *client.MapVerifier,
-	logVerifier client.LogVerifier) *Verifier {
+	mapVerifier *tclient.MapVerifier,
+	logVerifier tclient.LogVerifier) *Verifier {
 	return &Verifier{
 		vrf:         vrf,
 		mapVerifier: mapVerifier,
 		logVerifier: logVerifier,
 	}
 }
 
+// NewVerifierFromConfig creates a new instance of the client verifier from a config.
+func NewVerifierFromConfig(config *pb.Domain) (*Verifier, error) {
+	logVerifier, err := tclient.NewLogVerifierFromTree(config.GetLog())
+	if err != nil {
+		return nil, err
+	}
+
+	mapVerifier, err := tclient.NewMapVerifierFromTree(config.GetMap())
+	if err != nil {
+		return nil, err
+	}
+
+	// VRF key
+	vrfPubKey, err := p256.NewVRFVerifierFromRawKey(config.GetVrf().GetDer())
+	if err != nil {
+		return nil, fmt.Errorf("Error parsing vrf public key: %v", err)
```

Comment:
error values are supposed to start with lower-case so they can be composed together more readably.

Reply:
I wish there was an open source linter for this rule...